catalog/app/containers/Bucket/PackageDialog/PackageCreationForm.tsx

```diff
@@ -62,6 +66,66 @@ export interface PackageCreationSuccess {
   hash?: string
 }
 
+function createReadmeFile(name: string) {
+  const contents = [
+    `# ${name}`,
+    '\n\n',
+    `Stub README for the **${name}** package generated by Quilt Catalog`,
+    '\n',
+  ]
+  const f = new File(contents, README_PATH, { type: 'text/markdown' })
+  return FI.computeHash(f) as FI.LocalFile
+}
+
+// XXX: move to dialogs module
+interface DialogsOpenProps {
+  close: (reason?: string) => void
+}
+
+function ConfirmReadme({ close }: DialogsOpenProps) {
+  return (
+    <>
+      <M.DialogTitle>Add a README file?</M.DialogTitle>
+      <M.DialogContent dividers>
+        <M.DialogContentText>
+          You are about to push an empty package.
+          <br />
+          Would you like to add a stub <b>README.md</b> file?
+        </M.DialogContentText>
+      </M.DialogContent>
+      <M.DialogActions>
+        <M.Button onClick={() => close('cancel')} color="primary">
+          Cancel
+        </M.Button>
+        <M.Button onClick={() => close('empty')} color="primary" variant="outlined">
```

Comment:
So, as I see this feature, this third option is redundant:

Add a README file?
* Yes -> create empty (or not empty) package with empty README
* No/Cancel -> continue creating package

Reply:
in its current state it makes sense to keep this option